maturity-level-one/README.md

```diff
@@ -148,23 +148,21 @@ OpenAPI tooling, such as [AutoRest](https://github.com/Azure/autorest) & [NSwag]
 ### Generating OpenAPI Documentation
 Every API should have documentation in the OpenAPI format. If you want to generate those based on your code you can use tools like [Swashbuckle](https://github.com/domaindrivendev/Swashbuckle) & [NSwag](https://github.com/RSuter/NSwag).
 
-Naming convention for **OperationId** is composed of `{Controller}_{OperationName}`
```

Comment:
Why did we remove this?

Reply:
Because the same convention is highlighted two lines before